core/src/main/java/com/klaytn/caver/utils/AccountKeyPublicUtils.java

```diff
@@ -51,8 +51,8 @@ public static AccountKeyPublic decompressKey(String compressedPublicKey) {
         compEnc[0] = (byte)(yBit ? 0x03 : 0x02);
         ECPoint ecPoint = CURVE.getCurve().decodePoint(compEnc);
         return AccountKeyPublic.create(
-                Numeric.prependHexPrefix(ecPoint.getAffineXCoord().toString()),
-                Numeric.prependHexPrefix(ecPoint.getAffineYCoord().toString())
+                Numeric.prependHexPrefix(Numeric.toHexStringNoPrefixZeroPadded(ecPoint.getAffineXCoord().toBigInteger(),64)),
```

Comment:
Using toHexStringWithPrefixZeroPadded, you can also remove the call of `Numeric.prependHexPrefix`

Reply:
thanks, you're right. i'll reflect it.